pom.xml

```diff
@@ -176,6 +176,7 @@
     <maven-stylus-skin.version>1.5</maven-stylus-skin.version>
     <maven-surefire-plugin.version>3.0.0-M1</maven-surefire-plugin.version>
     <maven-checkstyle-plugin.version>3.1.2</maven-checkstyle-plugin.version>
+    <maven-assembly-plugin.version>3.4.1</maven-assembly-plugin.version>
```

Comment:
Sorry, I mean we should update the apache parent pom to version 27.  Then, we don't need to add maven-assembly-plugin.version here.  See below:
```
+++ b/pom.xml
@@ -18,7 +18,7 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>23</version>
+    <version>27</version>
     <relativePath /> <!-- resolve from repository -->
   </parent>
 
@@ -176,7 +176,6 @@
     <maven-stylus-skin.version>1.5</maven-stylus-skin.version>
     <maven-surefire-plugin.version>3.0.0-M1</maven-surefire-plugin.version>
     <maven-checkstyle-plugin.version>3.1.2</maven-checkstyle-plugin.version>
-    <maven-assembly-plugin.version>3.4.1</maven-assembly-plugin.version>
 
     <checkstyle.version>9.3</checkstyle.version>
@@ -522,7 +522,6 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-assembly-plugin</artifactId>
-          <version>${maven-assembly-plugin.version}</version>
           <configuration>
             <!--Defer to the ratis-assembly sub-module.  It does all assembly-->
             <skipAssembly>true</skipAssembly>
```

Reply:
Just tried it but there were some failures.  Let's don't update apache parent version for now.